package.json

```diff
@@ -12,6 +12,8 @@
     "postinstall": "rimraf node_modules/@types/mocha",
     "pretest": "rimraf coverage && lerna run build:i18n",
     "test": "jest --config=config/jest.config.js --coverage",
+    "test:cypress": "lerna run build && cypress run --component && lerna run clean",
+    "test:all": "yarn test:cypress && yarn test",
     "clean": "lerna run clean --stream && rimraf coverage",
```

Comment:
I would add another `"cypress:open": "cypress open"` script here

Reply:
Done. I set component testing and Chrome as default.